src/main/java/com/xceptance/neodymium/module/statement/testdata/TestdataStatement.java

```diff
@@ -266,6 +250,21 @@ else if (randomSetAmount > 0)
                 }
             }
         }
+        RandomDataSets methodRandomDataSetsAnnotation = method.getAnnotation(RandomDataSets.class);
```

Comment:
please ad somme comments for the random data set logic to help out our future me

Reply:
Done